Sources/Brave/Frontend/Shields/SubmitReportSuccessView.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import SnapKit
+
+struct SubmitReportSuccessView: View {
+  var body: some View {
+    VStack(alignment: .center, spacing: 16) {
+      Image(braveSystemName: "leo.check.circle-outline")
+        .resizable()
+        .frame(width: 48, height: 48)
```

Comment:
Don't use `resizable` & `frame` on an SF Symbol, switch to `font` with a specific font size that matches what you need

Reply:
This was my original intention but somehow felt like it was a hack but ok, this is now fixed.